imagevector/images.yaml

```diff
@@ -115,7 +115,7 @@ images:
 - name: cluster-autoscaler
   sourceRepository: github.com/gardener/autoscaler
   repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
-  tag: "v1.27.0"
+  tag: "v1.27.1"
```

Comment:
Is this tag published? I don't see any [release](https://github.com/gardener/autoscaler/tags) for it. Also docker pull is not working.

Reply:
WIth which PR we should proceed this one of https://github.com/gardener/gardener/pull/8588. If you want this PR please adapt the release notes properly so that it comes as autoscaler release notes.

Reply:
pls merge https://github.com/gardener/gardener/pull/8588 in that case, I'll close my PR